### PR TITLE
Added release notes changes

### PR DESCRIPTION
**Pull request by hardikhdholariya**

### CODE REVIEW

File: docs/release_notes/release_notes.md [hunk 1]
```diff
@@ -9,26 +9,91 @@ has_children: true
...
 
+* ### BlockShield Integration
+    <!-- TODO - Add detailed info on this section. -->
+    * Removed everything related to honeyDB, blocked IPs.
+
+* ### F5 BIGIP
+    * Added new dashboard named **F5 BIGIP ASM**.
+    * Added new alert named **F5 BIGIP - Not Blocked Attacks**.
+
+* Added new alerts for the Sophos Firewall:
```
Comment (VatsalJagani) [inline docs/release_notes/release_notes.md:28]: Make Sophos Firewall in bold

File: docs/release_notes/release_notes.md [hunk 2]
```diff
@@ -9,26 +9,91 @@ has_children: true
...
+    * Sophos Firewall - Lost Connection to Sophos Central
+    * Sophos Firewall - VPN Tunnel Down
+    * Sophos Firewall - Gateway Down
+    * Sophos Firewall - Advanced Threat Detected
+
+* Added new alerts for the MSSQL and Oracle databases:
```
Comment (VatsalJagani) [inline docs/release_notes/release_notes.md:34]: MSSQL Database and Oracle Database
* and in bold text

File: docs/release_notes/release_notes.md [hunk 3]
```diff
@@ -9,26 +9,91 @@ has_children: true
...
+    * MSSQL - Database Changes
+    * MSSQL - Role Changes
+    * Oracle - Database Changes
+    * Oracle - Role Changes
+
+* Added new alerts for the Defender ATP:
```
Comment (VatsalJagani) [inline docs/release_notes/release_notes.md:40]: Defender ATP in bold text.

File: docs/release_notes/release_notes.md [hunk 4]
```diff
@@ -9,26 +9,91 @@ has_children: true
...
 * Removed the following deprecated alerts:
     * O365 - Login Failure Outside Home Country Due To Multi Factor Authentication
     * O365 - Successful Login Outside Home Country
     * Authentication - Successful VPN Login Outside Home Country
     * Linux - Change in Sudo Access of Local Linux Account
 
-* Removed the [Splunk Add-on for RWI - Executive Dashboard](https://splunkbase.splunk.com/app/5063/) dependency.
+* Removed the following other app dependent macros and defined related macro in app itself:
+    * Added **cs_drop_dm_object_name** macro to replace the **drop_dm_object_name** macro.
+    * Added **cs_cim_authentication_indexes** macro to replace the **cim_Authentication_indexes** macro.
```
Comment (VatsalJagani) [inline docs/release_notes/release_notes.md:54]: Shall we name the index just cs_authentication_indexes (as now we are not even using CIM data model)

File: docs/release_notes/release_notes.md [hunk 5]
```diff
@@ -9,26 +9,91 @@ has_children: true
...
+* Added "Object Type" and "Object Name" filter on **MSSQL** and **Oracle** dashboards.
 
 * Removed the following deprecated alerts:
     * O365 - Login Failure Outside Home Country Due To Multi Factor Authentication
     * O365 - Successful Login Outside Home Country
     * Authentication - Successful VPN Login Outside Home Country
     * Linux - Change in Sudo Access of Local Linux Account
 
-* Removed the [Splunk Add-on for RWI - Executive Dashboard](https://splunkbase.splunk.com/app/5063/) dependency.
+* Removed the following other app dependent macros and defined related macro in app itself:
```
Comment (VatsalJagani) [inline docs/release_notes/release_notes.md:52]: No need to mention this point to release notes actually.
For now move it at the end in a section called **### For Splunk Admins**

File: docs/release_notes/release_notes.md [hunk 6]
```diff
@@ -9,26 +9,91 @@ has_children: true
...
+
+    * Fixed the typo in the macro name from **cs_authentication_vpn_login_attemps_outside_working_hour_filter** to **cs_authentication_vpn_login_attempts_outside_working_hour_filter**
 
 
 
 ## Upgrade Guide from 4.9.0 to 5.0.0
+    
+* After upgrade, only SOC related alerts will be received to existing configured critical emails. To make more changes, configure the SOC and Compliance teams related configs under **Cyences Settings > Cyences App Configuration > Cyences Alerts Configuration** section.
+
+* In order to use the sophos firewall alerts, onboard the **sophos_events** data from [Sophos Central Addon for Splunk](https://splunkbase.splunk.com/app/6186/).
```
Comment (VatsalJagani) [inline docs/release_notes/release_notes.md:97]: Add link to data onboarding page from our document.

**Review by VatsalJagani — COMMENTED** [review]
Have we covered already merged commits to master till now??

Have we looked at the 5.0.0-changes branch commits to add to release notes?

**Comment by VatsalJagani** [comment]
@hardikhdholariya  - We also need to include all the doc changes necessary for this stuff.

File: docs/release_notes/release_notes.md (hunk 4 above)
Comment (hardikhdholariya) [reply docs/release_notes/release_notes.md:54]: we already have cs_authentication_indexes macro which we are using in normal and tstats searches.
& cs_cim_authentication_indexes macro we are using for the Authentication datamodel base search only.

so may be we can rename it to cs_dm_authentication_indexes or do you want to use common macro cs_authentication_indexes everywhere?

Comment (VatsalJagani) [reply docs/release_notes/release_notes.md]: cs_cim_authentication_indexes OR cs_dm_authentication_indexes -> I don't see it.

**Pull request merged**